game/scripts/vscripts/gamemode.lua

```diff
@@ -30,13 +30,13 @@ require('libraries/notifications')
 -- This library can be used for starting customized animations on units from lua
 require('libraries/animations')
 -- This library can be used for performing "Frankenstein" attachments on units
-require('libraries/attachments')
```

Comment:
Don't we use this? I thoughts that's how to attach stuff to hero models and stuff...